api/write-raw.md

```diff
@@ -0,0 +1,116 @@
+# Writing Raw Data
+
+In contrast to the other submission APIs ([numeric](./write-nnt.md),
+[text](./write-text.md), [histogram](./write-histogram.md)), which accept
```

Comment:
histograms and text aren't pre-rolled.  Just NNT... I would say specifically-typed data ingestion.